stacks/cloud/README.md

```diff
@@ -17,45 +17,88 @@ On Windows, the hosts file is usually located under `c:\Windows\System32\Drivers
 
 The stack uses multiple compose files to setup the services and reach the desired state. Common definitions to both development and production settings are grouped in the `base` directory and specific ones are grouped in the `dev` and `prod` directories. For example, when configuring `traefik` as service's load balancer, you can deploy its development definition to avoid exposing services without need. Also, it can be necessary to deploy the services into multiple machines, which could be done by using the `multi-node` definitions.
 
-### Development
+Furthermore, when deploying the stack, you have the following options:
+
+- **(1)** Deploy stack as a **cloud** instance;
+- **(2)** Deploy stack as a **fog** instance and connecting to another, remote, _cloud_ instance;
+- **(3)** Deploy stack as both a **cloud** and **fog** instance.
 
-After using the `init` command as described [here](../../README.md#Development), the stack files will be copied to `<path>/stack` directory (`path` is the folder specified in the `init` execution), be sure to enter there. The following files should be shown when exploring the `stack` directory
+When following the instructions from here and on, you will **(1) deploy stack as a fog instance and connecting to another, remote, _cloud_ instance** (in our case, KNoT's testing instance, located in a remote cloud provider server). If you're weary, and want to check if KNoT's cloud instance is up and running before following to the next steps, just do:
 
 ```bash
-cd <path>/stack
-ls
+$ curl https://api.knot.cloud/healthcheck
+{"status":"online"}
 ```
 
-```text
+>_**NOTE**: If the command above does not return the expected output, it likely means the KNoT Cloud testing instance if offline. In this case, please reach out to us in our [Slack Workspace](https://join.slack.com/t/knot-iot/shared_invite/zt-8tgbqqon-WUZ_54_A~i~zle~JBk20Kg)._
+
+To **(2) deploy stack as a cloud instance**,
+?? -> <context>  <rename from `*.fog` to `*.cloud`> <stack can be used as a fog instance when deployed with a connector addon>
```

Comment:
To deploy as a simple cloud instance `(1)`, we can only suggest the user rename the DNS for avoiding confusion and ignore the steps related to connection deployment and attachment to the remote instance.

Reply:
For this option, there's no `connector` involved.

Reply:
Done :smile: